package.json

```diff
@@ -67,6 +67,7 @@
     "cytoscape": "^3.3.0",
     "cytoscape-cose-bilkent": "^4.0.0",
     "cytoscape-dagre": "^2.2.2",
+    "embark": "4.0.0-alpha.2",
```

Comment:
Added Embark as a local dep so that we can have Embark 4 (or other)  as a global install, but it will still use the local alpha 2